subsys/bluetooth/host/scan.c

```diff
@@ -805,6 +805,14 @@ void bt_hci_le_per_adv_report(struct net_buf *buf)
 			 */
 			bt_hci_le_per_adv_report_recv(per_adv_sync, &buf->b, &info);
 		} else {
+			if (evt->data_status == BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS_INCOMPLETE) {
+				LOG_WRN("Received incomplete advertising data. "
+					"Advertising report dropped.");
```

Comment:
This condition is expected during normal operation due to noise in the environment, right? Then it should not be a warning.

Reply:
should it be replaced with a LOG_DBG?